packages/typography/README.md

```diff
@@ -0,0 +1,11 @@
+# Typography
+
+This package contains typographic elements used in WordPress.com products
+
+## Installation
+
+```sh
+yarn add @automattic/typography
+```
+
+## Usage
```

Comment:
That's the gist! In the future I could see us moving more shared typographic elements (type scale, mixins for font styling, etc.) into this package so I think we should keep the ability to compile Sass; otherwise, happy to lose the cruft.

Reply:
Even If the package distributes SASS files, it doesn't need to compile them down to CSS during package build. For example, compiling a SASS file with mixin or variable definitions compiles all the SASS syntax away and the result is an empty file.

It might be better to author the CSS files directly in CSS (for example, in the `fonts.css` file, the SASS syntax doesn't add _that_ much value). Then we don't need any build step, as the package is basically a folder with `*.css` and `*.scss` files that are distributed as-they-are.

Reply:
Aha, I get it! Let me rework this to use just Sass and CSS with a CSS module as an example...

Reply:
We might need to enable some CSS module features in our webpack config. Let me know if you run into any roadblocks. This stuff is new to me, too. I have only read about it until now 🙂

Reply:
I'd like eventually have most/all of this included in this package:

https://github.com/Automattic/wp-calypso/blob/9000f926ccc67af288604471d2ea53c0befd54de/client/assets/stylesheets/shared/_typography.scss

Reply:
I started playing around with this; it looks like we need to add `css-loader` and `style-loader` support to webpack config, according to everything I've read. I tinkered with it a bit but couldn't get it to work locally.

If we do this, we'd also need to differentiate between cases where we want to load CSS as a module vs. loading global CSS (I'm not sure if we use any global *.css files in Calypso?)